### PR TITLE
feat(GCPVpcPeering) Adding documentation and tutorial for the feature

### PR DESCRIPTION
**Pull request by bru-jer-work**
**Description**

Changes proposed in this pull request:

- Creating documentation and tutorial for the GCP VPC Peering feature

**Related issue(s)**
Issues#453

### CODE REVIEW

File: docs/user/resources/04-50-gcp-vpc-peering.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+connection that can be created to allow communication between Kyma and a remote VPC in Google Cloud Platform (GCP).
+It allows you to consume services available on the remote VPC from the Kyma cluster.
+
+### Required permissions on the remote project
+In order to create a VPC peering connection, the following permissions are required and must be granted to the Kyma service account provided:
+
+| Parameter                          | Description                                                                 |
+|------------------------------------|-----------------------------------------------------------------------------|
+| compute.networks.addPeering        | Required to create the peering request on the remote project and VPC.       |
+| compute.networks.get               | Required to fetch the list of existing VPC peerings on the remote VPC.      |
```
Comment (nataliasitko) [inline docs/user/resources/04-50-gcp-vpc-peering.md:13]:
```suggestion
| compute.networks.get               | Required to fetch the list of existing VPC peerings from the remote VPC.      |
```

File: docs/user/tutorials/01-30-gcp-vpc-peering.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+This tutorial explains how to create a VPC peering connection between a remote VPC network and Kyma in Google Cloud Platform (GCP).
+
+## Prerequisites
+- You have the Cloud Manager module added.
+- Use a POSIX-compliant shell or adjust the commands accordingly. For example, if you use Windows, replace the `export` commands with `set` and use `%` before and after the environment variables names.
+
+## Steps <!-- {docsify-ignore} -->
+
+1. Fetch your Kyma ID
```
Comment (nataliasitko) [inline docs/user/tutorials/01-30-gcp-vpc-peering.md:11]:
```suggestion
1. Fetch your Kyma ID.
```

File: docs/user/tutorials/01-30-gcp-vpc-peering.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
+# Create Vpc Peering in GCP
+
+This tutorial explains how to create a VPC peering connection between a remote VPC network and Kyma in Google Cloud Platform (GCP).
+
+## Prerequisites
```
Comment (nataliasitko) [inline docs/user/tutorials/01-30-gcp-vpc-peering.md:5]:
```suggestion
## Prerequisites  <!-- {docsify-ignore} -->
```

File: docs/user/resources/04-50-gcp-vpc-peering.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# GcpVpcPeering Custom Resource
+
+The `gcpvpcpeering.cloud-resources.kyma-project.io` custom resource (CR) describes the VPC peering
+connection that you can create to allow communication between Kyma and a remote VPC in Google Cloud Platform (GCP).
+It allows you to consume services available on the remote VPC from the Kyma cluster.
+
+### Required Permissions on the Remote Project
```
Comment (nataliasitko) [inline docs/user/resources/04-50-gcp-vpc-peering.md:7]:
```suggestion
### Required Permissions on the Remote Project <!-- {docsify-ignore} -->
```
I noticed that you use `<!-- {docsify-ignore} -->` for all headings of the docs located in `/resources` and `/tutorials` folders, so let's add it also here

**Review by nataliasitko — APPROVED** [review]

**Review by ngrkajac — APPROVED** [review]

**Pull request merged**